custom/samveg/case_importer/validators.py

```diff
@@ -26,7 +26,7 @@
 )
 
 
-class RequiredColumnsValidator(BaseOperation):
+class RequiredColumnsValidator(object):
```

Comment:
nit: `(object)` is not needed since Python 3.
```suggestion
class RequiredColumnsValidator:
```